Swabbr.Core/Notifications/NotificationAction.cs

```diff
@@ -14,34 +14,19 @@ public enum NotificationAction
         /// </summary>
         FollowedProfileVlogPosted = 0,
 
-        /// <summary>
-        ///     Used to motivate the user to be more active.
-        /// </summary>
-        InactiveUserMotivate = 1,
-
-        /// <summary>
-        ///     Indicates the user has unwatched new vlogs.
-        /// </summary>
-        InactiveUnwatchedVlogs = 2,
-
-        /// <summary>
-        ///     Indicates the user should record more vlogs.
-        /// </summary>
-        InactiveVlogRecordRequest = 3,
-
         /// <summary>
         ///     Indicates a user vlog gained a like.
         /// </summary>
-        VlogGainedLikes = 4,
+        VlogGainedLike = 1,
```

Comment:
We have only 4 notification cases left instead of the original 8. Since we have the luxury of being able to remove anything we don't use across both app and backend I chose to do so.